lib/hodor/cli.rb

```diff
@@ -1,30 +1,27 @@
 require "thor/runner"
-require_relative 'environment'
 
-module Hodor
```

Comment:
This is just white space changes, making it seem so large.

All I did here was translate ```module Hodor module Cli``` to ```module Hodor::Cli```